dakara_server/users/migrations/0001_initial.py

```diff
@@ -17,22 +17,147 @@ class Migration(migrations.Migration):
+                        verbose_name='email address'
+                    )
+                ),
+                (
+                    'is_staff',
+                    models.BooleanField(
+                        help_text=('Designates whether the user can log into '
+                                   'this admin site.'),
+                        default=False,
+                        verbose_name='staff status')),
+                (
+                    'is_active',
+                    models.BooleanField(
+                        help_text=('Designates whether this user should be '
+                                   'treated as active. Unselect this instead '
+                                   'of deleting accounts.'),
+                        default=True,
+                        verbose_name='active')),
+                (
+                    'date_joined',
+                    models.DateTimeField(
+                        default=django.utils.timezone.now,
+                        verbose_name='date joined'
+                    )
+                ),
+                (
+                    'users_permission_level',
+                    models.CharField(
+                        max_length=1,
+                        null=True,
+                        choices=[('u', 'User'), ('m', 'Manager')])
+                ),
+                (
+                    'library_permission_level',
+                    models.CharField(
+                        max_length=1,
+                        null=True,
+                        choices=[('u', 'User'), ('m', 'Manager')])
+                ),
+                (
+                    'playlist_permission_level',
+                    models.CharField(
+                        max_length=1,
+                        null=True,
+                        choices=[('p', 'Player'), ('u', 'User'),
+                                 ('m', 'Manager')])
+                ),
+                (
+                    'groups',
+                    models.ManyToManyField(
+                        to='auth.Group',
+                        verbose_name='groups',
+                        related_query_name='user',
+                        help_text=('The groups this user belongs to. A user '
+                                   'will get all permissions granted to each '
+                                   'of their groups.'),
+                        blank=True,
+                        related_name='user_set')
+                ),
+                (
+                    'user_permissions',
+                    models.ManyToManyField(
+                        to='auth.Permission',
+                        verbose_name='user permissions',
+                        related_query_name='user',
+                        help_text='Specific permissions for this user.',
+                        blank=True,
+                        related_name='user_set'
+                     )
+                ),
```

Comment:
this file was auto-generated, does it really need to comply with PEP8 ?

Reply:
Bypassing is somewhat [buggy](https://gitlab.com/pycqa/flake8/issues/393) for commit hook.